### ISSUE TITLE
How to Create subset using mdx string

### ISSUE THREAD

**Issue by reddibhatini** [cubewise-code/tm1py#206]
Hello,

Is there a way to create a subset using MDX? Could not able to find a method for this.

Thank you

**Comment by rclapp** [comment]
I have a Method waiting on a pull request that lets you execute an MDX query against a dimension. A complimentary function that saves it as a subset would be nice.

Thanks,

Ryan Clapp

**Comment by MariusWirtz** [comment]
Hi @reddibhatini,
here is a sample:

``` python
with TM1Service(address=ADDRESS, port=PORT, user=USER, password=PASSWORD, ssl=SSL) as tm1:

    dimension_name = "}Clients"
    expr = "{TM1SubsetAll([}Clients])}"

    subset = Subset(
        subset_name="All Clients",
        dimension_name="}Clients",
        hierarchy_name="}Clients",
        expression=expr)
    if not tm1.dimensions.hierarchies.subsets.exists(
            subset_name=subset.name,
            dimension_name=dimension_name,
            hierarchy_name=dimension_name,
            private=False):
        tm1.dimensions.hierarchies.subsets.create(subset, private=False)
    else:
        tm1.dimensions.hierarchies.subsets.update(subset, private=False)
```
@rclapp 
Thanks for the MR. Much appreciated! Will look into it in more depth on the weekend. 
I agree. It would be nice to have a simplified method to create a subset from mdx with just one call.

**Comment by reddibhatini** [comment]
@rclapp @MariusWirtz Thank you. That was helpful.
Yeah. I was looking for method call to create subset by mdx.
